src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUnitTest.java

```diff
@@ -1,13 +1,225 @@
  * Created by jonn on 8/22/17.
  */
 public class FuncotatorUnitTest extends GATKBaseTest {
 
+    /**
+     * Data provider for the unit test for checkIfAlreadyAnnotated. In this case, we have a
+     * "Funcotator" column in the vcf header, which indicates
+     * that this vcf has already been annotated and thus the program
+     * should run on this input.
+     *
+     */
+    @DataProvider
+    Object[][] provideDataForAnnotationCheckWrong() {
+        final Set<VCFHeaderLine> headerLines = new LinkedHashSet<>();
+
+        headerLines.add(new VCFHeaderLine("fileformat", "fileformat"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FILTER", "FILTER"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("FORMAT", "FORMAT"));
+        headerLines.add(new VCFHeaderLine("Funcotator", "Funcotator"));
```

Comment:
Add a second test case with a header containing a "Funcotator Version" line, but not a "Funcotator" line, to exercise the `else if` clause in the method you're testing.